setup.py

```diff
@@ -51,16 +51,16 @@
     "author": "C. Titus Brown",
     "author_email": "titus@idyll.org",
     "license": "BSD 3-clause",
-    "packages": ["sourmash_lib"],
+    "packages": find_packages(),
```

Comment:
Presumably recursively finds all packages and subpackages?

Reply:
Yes, it is recommended by the Python packaging guide: https://packaging.python.org/tutorials/distributing-packages/#packages